notebooks/data_table_transformer/main.py

```diff
@@ -0,0 +1,125 @@
+import os
```

Comment:
I recommend adding authorship -- it will be useful for users as well as Terra user support to know who to contact if issues come up. You could also link to bdcat-featured-notebooks repo if folks want to contribute?

Reply:
`+ 1`

Reply:
Will do!